keylime/tenant.py

```diff
@@ -662,7 +662,7 @@ def do_cv(self):
 
         if response.status_code == 409:
             # this is a conflict, need to update or delete it
-            logger.error("Agent %s already existed at CV. Please use delete or update.", self.agent_uuid)
+            logger.error(json.dumps(response.json()))
```

Comment:
Is there a key in the json payload that has the error message? Iss this dumping a data structure?

Reply:
@mpeters Yes - it's just printing out the entire JSON response from the verifier, wich includes the response code and error message. This is similar to how the [allowlist API commands](https://github.com/keylime/keylime/blob/master/keylime/tenant.py#L1277) handle responses without doing any parsing.

In general, the tenant's response handling is pretty inconsistent. Some commands (including the one I'm replacing) parse the response or response code in some form to deliver tailored output to the user, while others (notably the allowlist API) just forward whatever output came from the verifier directly.

I can tidy that up in a follow-up PR if we want.

Reply:
Update - in hindsight, realized it was silly to not use the existing `_print_json_response` function if that's more or less all I'm doing, so that's been changed.